Dockerfile

```diff
@@ -18,29 +20,28 @@ RUN \
   apk add --no-cache \
     fontconfig \
     memcached \
-    php8-ctype \
-    php8-curl \
-    php8-dom \
-    php8-gd \
-    php8-ldap \
-    php8-mbstring \
-    php8-mysqlnd \
-    php8-openssl \
-    php8-pdo_mysql \
-    php8-pecl-memcached \
-    php8-phar \
-    php8-simplexml \
-    php8-tokenizer \
+    php81-ctype \
+    php81-curl \
+    php81-dom \
+    php81-gd \
+    php81-ldap \
+    php81-mbstring \
+    php81-mysqlnd \
+    php81-openssl \
+    php81-pdo_mysql \
+    php81-pecl-memcached \
+    php81-phar \
+    php81-simplexml \
+    php81-tokenizer \
     qt5-qtbase \
-    tar \
     ttf-freefont && \
   apk add --no-cache \
     --repository=http://dl-cdn.alpinelinux.org/alpine/v3.14/community \
     wkhtmltopdf && \
```

Comment:
This won't run any more because the 3.14 deps are too different, there's been ongoing conversation about options around it but right now it's not usable even on the existing 3.15 image.

Reply:
Resolved in https://github.com/linuxserver/docker-bookstack/pull/158/commits/1603fb13fcf14f0120b03b29dc13a45f0d5d69f5